pkg/tcp/proxy_test.go

```diff
@@ -79,3 +82,83 @@ func TestCloseWrite(t *testing.T) {
 	require.Equal(t, int64(4), n)
 	require.Equal(t, "PONG", buffer.String())
 }
+
+func mustHaveExternalNetwork(t *testing.T) {
+	if runtime.GOOS == "js" {
```

Comment:
What is it for?

Reply:
To make sure we don't try to run this test in an OS that doesn't support external networks